apps/epic-react/src/templates/lesson-template.tsx

```diff
@@ -84,6 +84,7 @@ const ExerciseTemplate: React.FC<{
   const displayLessonCompletionToggle =
     (lesson._type === 'solution' ||
       lesson._type === 'explainer' ||
+      lesson._type === 'exercise' ||
```

Comment:
The lessons wrapped in sections have `exercise` type (excluding `intro`/`outro` that have `explainer` type) so I've added this type to condition and now completion toggle is shown.

Reply:
I think we want them to be able to "toggle complete" any lesson type. Does this cover all lesson types?

Reply:
I believe yes